core/src/smartcontracts/isi/triggers.rs

```diff
@@ -1,22 +1,518 @@
 //! This module contains implementations of smart-contract traits and
 //! instructions for triggers in Iroha.
 
-use iroha_data_model::{
-    error::{FindError, MathError},
-    prelude::*,
-};
+use iroha_data_model::{isi::error::MathError, prelude::*, query::error::FindError};
 use iroha_telemetry::metrics;
 
+pub mod set {
```

Comment:
If we restrict this module to `pub(crate)` we can reduce the binary size of `iroha_core`. Given that almost everything that it can depend on is in `core` anyway, restricting visibility will not take much effort.

Reply:
how would changing visibility that reduce the size?

Reply:
nvm, done it